### PR TITLE
highlighted the end scores using box shadow and background color

### PR DESCRIPTION
**Pull request by palaksv**
#24

### CODE REVIEW

**Comment by error401de** [comment]
Thanks for participating @palaksv!
I fear I wasn't clear enough in my description in #24: Goal was to highlight it _depending_ on the score. So e.g. win = green background, loss = red background and even = yellow one.
Want to try that?

**Comment by palaksv** [comment]
Yeah, sure

**Comment by error401de** [comment]
replaced by #24
